client/src/components/ProgressIndicator/index.tsx

```diff
@@ -0,0 +1,282 @@
+    const result = (completed / length) * 100;
+
+    if (!result) {
+      return 0;
+    }
+    if (result < 1) {
+      return Number(result.toFixed(2));
+    }
+    return Math.floor(result);
+  };
+
+  const completedChallengePercentage = computePercentage({
+    completed: completedChallengeCount,
+    length: allChallengeCount
+  });
+
+  const completedCertificatePercentage = computePercentage({
+    completed: earnedCertificateCount,
+    length: allCertificateCount
+  });
+
+  /*
+   *
+   * Superblock progress
+   *
+   */
+
+  const getCompletedChallengesCount = ({
+    completedChallengesList = [],
+    superBlockChallengesList = []
+  }: {
+    completedChallengesList: CompletedChallenge[];
+    superBlockChallengesList: NodeChallenge[];
+  }) => {
+    const completedChallengesIDs = completedChallengesList.map(challenge => {
+      return challenge.id;
+    });
+
+    const completedChallengeObjects = superBlockChallengesList.filter(
+      challenge => {
+        return completedChallengesIDs.includes(challenge.node.challenge.id);
+      }
+    );
+
+    return completedChallengeObjects.length;
+  };
+
+  const getSuperBlockCompletedProjectsCount = ({
+    completedChallengesList = []
+  }: {
+    completedChallengesList: CompletedChallenge[];
+  }) => {
+    // Collect the id of challenges with challengeType === 14.
+    // Not sure yet if all challenges with challengeType of 14 are all projects.
+    // Mark null to id (item) without the challengeType property.
+    const completedProjectObjects = completedChallengesList.map(challenge => {
+      return challenge.challengeType && challenge.challengeType === 14
+        ? challenge.id
+        : null;
+    });
+    // Remove those items (id) which is null.
+    const completedProjectIDs = completedProjectObjects.filter(
+      id => id !== null
+    );
+
+    // Filter the projects for the current superblock.
+    const completedSuperBlockProjects = superBlockChallengesList.filter(
+      challenge => {
+        return completedProjectIDs.includes(challenge.node.challenge.id);
+      }
+    );
+
+    return completedSuperBlockProjects.length;
+  };
+
+  const superBlockCompletedChallengesCount = getCompletedChallengesCount({
+    completedChallengesList,
+    superBlockChallengesList
+  });
+  const superBlockCompletedChallengesPercent = computePercentage({
+    completed: superBlockCompletedChallengesCount,
+    length: superBlockTotalChallengesCount
+  });
+
+  const superBlockCompletedProjectsCount = getSuperBlockCompletedProjectsCount({
+    completedChallengesList
+  });
+  const superBlockCompletedProjectsPercent = computePercentage({
+    completed: superBlockCompletedProjectsCount,
+    length: superBlockTotalProjectsCount
+  });
+
+  const headerText = i18nSuperBlock ? i18nSuperBlock : 'Overall';
+
+  return (
+    <div className='progress-summary'>
+      <h3 className='progress-summary__main-header'>
+        {headerText} Progress Summary
+      </h3>
```

Comment:
```suggestion
      <h2 className='progress-summary__main-header'>
        {headerText} Progress Summary
      </h2>
```

Now that the progress summary is always above the certifications/courses, it needs to be an `h2` heading to fit into the hierarchy properly.